ACHNBrowserUI/ACHNBrowserUI/views/critters/ActiveCrittersView.swift

```diff
@@ -56,8 +59,9 @@ struct ActiveCrittersView: View {
     
     var body: some View {
         List {
-            if (viewModel.crittersInfo[.fish]?.toCatch.isEmpty == true || viewModel.crittersInfo[.bugs]?.toCatch.isEmpty == true) &&
+            if (viewModel.crittersInfo[.fish]?.toCatchLater.isEmpty == true || viewModel.crittersInfo[.bugs]?.toCatchLater.isEmpty == true) &&
                 (viewModel.crittersInfo[.fish]?.caught.isEmpty == true || viewModel.crittersInfo[.bugs]?.caught.isEmpty == true) {
+                // FIXME: update this condition for .toCatchNow + .toCatchLater
```

Comment:
You already updated for `.toCatchLater`. IMO you simply need to add
```
if (viewModel.crittersInfo[.fish]?.toCatchNow.isEmpty == true || viewModel.crittersInfo[.bugs]?.toCatchNow.isEmpty == true) &&
```
I would add this before the `.toCatchLater` line.